frame/lending/src/tests.rs

```diff
@@ -463,7 +460,7 @@ fn test_vault_market_cannot_withdraw() {
 
 		// We don't even wait 1 block, which mean the market couldn't withdraw funds.
 		assert_noop!(
-			Lending::borrow_internal(&market, &ALICE, MockCurrencyId::BTC.unit::<Balance>()),
```

Comment:
is value deposited in new check same as old ?

Reply:
not sure i got question, but in short - no units in lending protocol.